datalad_service/tasks/validator.py

```diff
@@ -23,9 +24,12 @@ def validate_dataset_sync(dataset_path):
     Runs the bids-validator process and installs node dependencies if needed.
     """
     setup_validator()
-    process = subprocess.run(
-        ['./node_modules/.bin/bids-validator', '--json', dataset_path], stdout=subprocess.PIPE)
-    return json.loads(process.stdout)
+    try:
+        process = subprocess.run(
+            ['./node_modules/.bin/bids-validator', '--json', dataset_path], stdout=subprocess.PIPE, timeout=60)
```

Comment:
FWIW, may be my hardware is subpar, but I did see bids-validator runs longer than a minute.  If the choice arbitrary I would have made it more generous (e.g. `600` or even more)